packages/block-editor/src/components/block-icon/stories/index.story.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * Internal dependencies
+ */
+import BlockIcon from '../';
+
+/**
+ * WordPress dependencies
+ */
+import { paragraph } from '@wordpress/icons';
+
+const meta = {
+	title: 'BlockEditor/BlockIcon',
+	component: BlockIcon,
+	parameters: {
+		docs: { canvas: { sourceState: 'shown' } },
+	},
+	argTypes: {
+		icon: {
+			control: 'block',
```

Comment:
There is no such [control.type](https://storybook.js.org/docs/api/arg-types#controltype) as `block`. A better solution would be to use an SVG icon of the actual block via the `select` control, similar to what we did with [the `BlockCard` component](https://github.com/WordPress/gutenberg/blob/80e79916a27f6142bb4d1b035875a57b4b74da0f/packages/block-editor/src/components/block-card/stories/index.story.js#L38-L52).